scripts/link-data.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+# This script is used to establish symlinks for training modules to use data
+# stored in a shared directory.
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+# move back up to the training modules root
+cd ..
+
+# location for the data modules
+share_base=/shared/data
+modules_base=${share_base}/training-modules
+
+
+# RNA-seq module directories
+mkdir -p RNA-seq/data/gastric-cancer
+mkdir -p RNA-seq/data/gastric-cancer/salmon_quant
+mkdir -p RNA-seq/data/NB-cell
+mkdir -p RNA-seq/QC/gastric-cancer/fastp/
+mkdir -p RNA-seq/QC/gastric-cancer/fastqc/
+
+
+
+link_locs=(
+  RNA-seq/data/gastric-cancer/gastric-cancer_metadata.tsv
+  RNA-seq/data/gastric-cancer/fastq
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585571
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585572
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585573
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585574
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585575
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585576
+  RNA-seq/data/gastric-cancer/salmon_quant/SRR585577
+  RNA-seq/QC/gastric-cancer/fastp/SRR585571
+  RNA-seq/QC/gastric-cancer/fastp/SRR585572
+  RNA-seq/QC/gastric-cancer/fastp/SRR585573
+  RNA-seq/QC/gastric-cancer/fastp/SRR585574
+  RNA-seq/QC/gastric-cancer/fastp/SRR585575
+  RNA-seq/QC/gastric-cancer/fastp/SRR585576
+  RNA-seq/QC/gastric-cancer/fastp/SRR585577
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585571
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585572
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585573
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585574
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585575
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585576
+  RNA-seq/QC/gastric-cancer/fastqc/SRR585577
+  RNA-seq/data/NB-cell/NB-cell_metadata.tsv
+  RNA-seq/data/NB-cell/salmon_quant
+)
+for loc in ${link_locs[@]}
+do
+  ln -ns ${modules_base}/${loc} ${loc}
```

Comment:
I debated this, and had it there in an earlier version. It makes things cleaner if you run the script more than once, but I was worried about inadvertent data loss, so I removed it.

Reply:
When would we expect inadvertent data loss to happen? (Famous last words I know 😬 ) I would expect that we'd have to run this multiple times as it gets updated for development (and I just ran it multiple times for review because I wasn't sure if the "missing" files were because the symlink didn't get created when the volume was full).

Reply:
Say I did some processing on my own and made a `RNA-seq/QC/gastric-cancer/fastp/SRR585572` folder in my repo. The link would then replace that folder, and I would be SOL. 

Not that I should be doing that, but it could happen? Scripts with `-f` in them scare me.

Reply:
Okay, see what you think of what I did in 2270d2b

Reply:
"The team won, David!" - us at every demo day